chutney/server/src/test/java/blackbox/SecuredControllerSpringBootIntegrationTest.java

```diff
@@ -204,44 +220,70 @@ private static Object[] securedEndPointList() {
 
     private static Object[] unsecuredEndPointList() {
         return new Object[][]{
-            {GET, "/api/v1/user", null, null, OK},
-            {POST, "/api/v1/user", null, "{}", OK},
-            {GET, "/api/v1/ui/plugins/linkifier/", null, null, OK},
+            {GET, "/api/v1/user", "AUTHENTICATED", null, OK},
+            {POST, "/api/v1/user", "AUTHENTICATED", "{}", OK},
+            {GET, "/api/v1/ui/plugins/linkifier/", "AUTHENTICATED", null, OK},
             {GET, "/api/v1/info/build/version", null, null, OK},
             {GET, "/api/v1/info/appname", null, null, OK},
-            {GET, "/api/v2/features", null, null, OK},
+            {GET, "/api/v2/features", "AUTHENTICATED", null, OK},
+            {GET, "/api/v1/sso/config", null, null, OK},
         };
     }
 
     @ParameterizedTest
     @MethodSource({"securedEndPointList", "unsecuredEndPointList"})
     public void secured_api_access_verification(HttpMethod httpMethod, String url, String authority, String content, HttpStatus status) throws Exception {
-        UserDto user = new UserDto();
-        user.setName("userName");
-        ofNullable(authority).ifPresent(user::grantAuthority);
+        UserDto userDto = new UserDto();
+        userDto.setName("admin");
+        userDto.setId("admin");
         MockHttpServletRequestBuilder request = request(httpMethod, url)
             .secure(true)
-            .with(user(user))
             .contentType(MediaType.APPLICATION_JSON);
         if (content != null) {
             request.content(content);
         }
+        ofNullable(authority).ifPresent(right -> {
+            if (!"AUTHENTICATED".equals(authority)) {
```

Comment:
Could we refactor this into a method to be called from this method and the following ?